### PR TITLE
[FIX] Draft message do not go away when whole message is removed

### LINKED ISSUE #923: Draft Message do not go away when you remove the whole message
- Your Rocket.Chat Experimental app version: 1.14.0.8021
- Your Rocket.Chat server version: ####
- Device (or Simulator) you're running with: ####

go to any channel
write something on message box
go out of channel
now go back and remove the whole saved message 
go back 
go to channel again the removed message comes back

### PR DESCRIPTION
**Pull request by weijia-yu**
@RocketChat/ReactNative

**Acceptance Criteria**

In one room, after typing "test", and go back, then enter the room, remove "test" message and go back. Then enter the room, it will be empty inputfiled

Closes #923

### CODE REVIEW

**Comment by weijia-yu** [comment]
@diegolmello can you please review it?

File: app/views/RoomView/index.js [hunk 1]
```diff
@@ -227,7 +227,7 @@ export default class RoomView extends React.Component {
 	componentWillUnmount() {
 		this.mounted = false;
 		const { editing, replying } = this.props;
-		if (!editing && this.messagebox && this.messagebox.current && this.messagebox.current.text) {
+		if (!editing && this.messagebox && this.messagebox.current) {
```
Comment (pranavpandey1998official) [inline app/views/RoomView/index.js]: are u sure this is needed `&& this.messagebox && this.messagebox.current`

Comment (IlarionHalushka) [reply app/views/RoomView/index.js]: it is necessary. If `this.messagebox.current` is undefined then on next line `const { text } = this.messagebox.current;` an error will be thrown: 'cannot destructure value text of undefined'

**Comment by IlarionHalushka** [comment]
@weijia-yu please follow this [ link ](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative?pullRequest=965)and accept the license

**Pull request merged**